src/App.css

```diff
@@ -14,3 +49,17 @@
 .dg_icon-link:hover .dg_popular_star {
   color: white;
 }
+
+.dg_section {
+  padding: 0;
+  position: relative;
+}
+
+@media screen and (max-width: 768px) {
+  .dg_dynamic_search_container {
+    display: inline;
+  }
+  .col-3 {
```

Comment:
we shouldn't need to add responsive styles, bootstarp can accomodate this, see here https://getbootstrap.com/docs/4.0/layout/grid/

more specifically, the original changes that were submitted to do not handle responsive ness

a combination of somethign more like

```jsx
<div className="row">
  <div className="col-md-3 col-xs-12">filters go here</div>
  <div className="col-md-9 col-xs-12">filters go here</div>
</div>
```

I have no clue what this looks like, but this should get you closer.

Then you can remove any of these media query styles.

Reply:
Yup that handles it. Cleaned up the css as well.